src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -115,4 +117,19 @@ public void addMessage(List<Message> messages, Entity entity, String user) {
   return messages;
  }
 
+  /** 
+   * Gets all the users
```

Comment:
End the comment in a period as though it is a full sentence
https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment